### DISCUSSION TITLE
关于在 Ghost 插入后的大小

### DISCUSSION THREAD

**Discussion by ShadowsSide** [walinejs/waline#1323] [Q&A] [answered]
作者你好,我想咨询以下 Waline 在插入时如何做到像您官网一样设置一定的宽度,而不是直接满屏显示. 以下是我博客插入后的图片.
![image](https://user-images.githubusercontent.com/61958350/187063001-0cd8e21f-7c6d-458f-8200-89c189ec0d30.png)

想要实现的效果:
![image](https://user-images.githubusercontent.com/61958350/187063049-bb31ceb0-a04f-4eb9-96b1-58acd861ce7a.png)

**Comment by lizheming** [comment, marked as answer]
一般是建议插入到对应大小的 DOM 容器内即可，也可以自己通过 CSS 控制。

**Comment by ShadowsSide** [comment]
谢谢回复,我去尝试一下~
